authentik/tenants/management/commands/ensure_template_schema.py

```diff
@@ -0,0 +1,12 @@
+"""Run bootstrap tasks"""
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
```

Comment:
I think this might be a better fit for a system migration instead of a management command, despite us saying we didn't want to use system migrations too much more